TESTS/mbed_platform/wait_ns/main.cpp

```diff
@@ -23,6 +23,11 @@
 #include "hal/us_ticker_api.h"
 #include "hal/lp_ticker_api.h"
 
+//FastModels not support timing test
+#if defined(__ARM_FM)
```

Comment:
I thought that specifying a specific target in tests was not recommended ?

Reply:
Correct is not.  This is however an architecture exclusion

@ARMmbed/mbed-os-hal Please review